NEWS.md

```diff
@@ -13,20 +13,24 @@ New Features:
 
 - The `keras` python module is exported
 
-- Introduced `%py_class%`, a new python class generator constructor.
+- Introduced `%py_class%`, a python class constructor constructor.
```

Comment:
I think the double `constructor` here is intentional, but it's hard to understand... Does this make sense: 

> an R-language constructor for Python classes

? Or similar?